test/test_clients.py

```diff
@@ -144,3 +144,32 @@ class MyClient(Client):
 
     assert app_one.session.id is None
     assert app_two.session.id is None
+
+@pytest.fixture(scope="function")
+def config_path():
+    return './wampy/testing/configs/crossbar.timeout.json'
+
+
+def test_respond_to_ping_with_pong(config_path, router):
```

Comment:
it's not really that clear from this testcase that the Pong is returned. However, looking at the setup i'm confident that it is. I think if i had to write a test for this i'd consider mocking out the Pong frame, or a method on it, and assert it was sent/called. the problem then is that the Pong won't actually be sent! So I've no better suggestion for you. Would you mind adding a brief comment to explain why an Exception isn't raised here, e.g. # server is conifgured to send a Ping at interval xxxx and without a Pong response from the wampy client, an Exception will raise and the client would disconnect.